Mathlib/Algebra/Homology/Homotopy.lean

```diff
@@ -495,14 +495,14 @@ def mkInductiveAux₂ :
 
 @[simp] theorem mkInductiveAux₂_zero :
     mkInductiveAux₂ e zero comm_zero one comm_one succ 0 =
-      ⟨0, zero ≫ (Q.xPrevIso rfl).inv, mkInductiveAux₂.proof_2 e zero comm_zero⟩ :=
+      ⟨0, zero ≫ (Q.xPrevIso rfl).inv, mkInductiveAux₂.proof_3 e zero comm_zero⟩ :=
```

Comment:
This is pretty scary, do we understand why this suddenly changes?

Reply:
Thanks for the suggestions. Here, I think it is related to the changes I have made in the `ComplexShape` file.